jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsSdkClient.kt

```diff
@@ -21,10 +20,6 @@ import software.aws.toolkits.core.utils.getLogger
 import software.aws.toolkits.core.utils.info
 
 class AwsSdkClient : Disposable {
-    init {
```

Comment:
Is the intent that the client manager disposes these?

Reply:
AwsSdkClient is an applicationService, so this is implicitly disposed when either IDE goes away, or plugin is uninstalled (if we ever get dynamic working...)

https://jetbrains.org/intellij/sdk/docs/basics/disposers.html#automatically-disposed-objects